freeipa.spec.in

```diff
@@ -248,6 +248,7 @@ Requires: %{name}-server-common = %{version}-%{release}
 Requires: %{name}-common = %{version}-%{release}
 Requires: python2-ipaclient = %{version}-%{release}
 Requires: python-ldap >= 2.4.15
+Requires: python-lxml
```

Comment:
You are misunderstanding the patch. odsmgr does **no longer** use ```python-lxml```. That's the whole point of the patch. ```xml.etree``` is part of Python's standard library.